src/applications/PseudoProgressionEstimator.cpp

```diff
@@ -842,51 +908,6 @@ bool PseudoProgressionEstimator::PseudoProgressionEstimateOnExistingModel(std::v
   return true;
 }
 
-VariableLengthVectorType PseudoProgressionEstimator::DistanceFunction(const VariableSizeMatrixType &testData, const std::string &filename, const double &rho, const double &bestg)
```

Comment:
[One of the errors on the latest build](https://dev.azure.com/CBICA/CaPTk/_build/results?buildId=1938&view=logs&j=dec1ec93-b88c-574e-65bb-00f3d1806dd6&t=df74820f-1014-5ef4-b628-0d2a059f97e6&l=873) can be possibly traced to here.